README.rst

```diff
@@ -1,5 +1,4 @@
-About nsupdate.info
-===================
+.. image:: logo/logotype_blue.png
```

Comment:
please no README.rst changes - we use this from setup.py and i suspect this won't work for pypi.org.